packages/expo-dev-client-components/src/useExpoTheme.ts

```diff
@@ -11,11 +11,11 @@ export function useCurrentTheme(): 'light' | 'dark' {
 
   let theme: 'light' | 'dark' = 'light';
 
-  if (preference !== 'no-preference') {
+  if (preference && preference !== 'no-preference') {
```

Comment:
just curious - was preference undefined somewhere?

Reply:
Yeah, preference sometimes was undefined or an empty string, so I just decided to rely on JS boolean coercion to check if it's there or not.